pytest_operator/plugin.py

```diff
@@ -746,11 +772,12 @@ async def track_model(
                None:  True if model_name exists on this controller
                False: create a new model and keep=False, unless keep=True explicitly set
                True:  connect to a model and keep=True, unless keep=False explicitly set
-        @param Optional[bool] keep:
-               None:  follows the value of use_existing
-               False: tracked model is destroyed at tests end
-               True:  tracked model remains once tests complete
-               --keep-models flag will override this options
+        @param Optional[ModelKeep, str, bool, None] keep:
+               ModelKeep  : See docs for the enum
+               str        : mapped to ModelKeep values
+               None       : Same as ModelKeep.IF_EXISTS
+               True       : Same as ModelKeep.ALWAYS
+               False      : Same as ModelKeep.NEVER, but respects keep-models flag
```

Comment:
I think the behavior has changed, right? 
```suggestion
               False      : Same as ModelKeep.NEVER, overrides the keep-models flag
```

Reply:
The previous `False` setting here would destroy models at the end of the test -- unless `keep-models` flag was set.  I have to maintain that `False` will repeat that behavior

Reply:
if you want to override the `keep-models` flag.  just use `ops_test.ModelKeep.NEVER`